.github/workflows/build.yaml

```diff
@@ -98,11 +100,21 @@ jobs:
       - name: Build App
         run: "yarn build --publish never -w ${{ matrix.build-args }}"
 
+      # TODO test installer
+      - name: Run tests
+        run: "yarn test"
+        timeout-minutes: 5
+        env:
+          ELEMENT_DESKTOP_EXECUTABLE: ${{ matrix.executable_path }}
+
       - name: Upload Artifacts
+        if: always()
```

Comment:
For my own understanding, why is this needed? It looks a bit odd to me

Reply:
So we always upload the executables, even if the test fails, so you can try the executable on your machine locally